src/pages/Editor.vue

```diff
@@ -1019,7 +1016,6 @@ export default {
         this.isImageUploaderDialogShown ||
         this.isPublishedPlioDialogShown ||
         this.isEmbedPlioDialogShown ||
-        this.pending ||
         this.isPlioPreviewShown
```

Comment:
Background disabling functionality has been moved and is handled by App.vue only right?
Even in Scorecard.vue, You've removed references to isBackgroundDisabled etc and only used showSpinner and hideSpinner.
Then why do we need background disabled here?